README.md

```diff
@@ -1 +1,59 @@
-# group-IV
+# LUNCHER - Bunch your lunch!
+
+If you are hungry, but not sure what to eat, using only one product from you resources you can get few delicious options for your lunch. Easy and fast recipes for every hungry individual.
+
+## Getting Started
+
+This app is designed for every individual who wish to have a good, fast and healthy lunch. With only one ingredient you can get a list of options and choose one that is the best.
+Enjoy.
+
+## Motivation
+
+A Ragnarson Internship 2016 group project: from theory to practice.
+
+## Built with
+
+* Ruby ```2.3.1``` on Rails ```5.0.0``` and gems:
+  * autoprefixer
+  * rspec
+* Bootstrap
+* postgreSQL
```

Comment:
you can drop this part, no reason to mention about every gem, they will be added all the time